src/components/atoms/atoms.css

```diff
@@ -0,0 +1,4 @@
+img {
+    width: 100%;
+    max-width: 100%;
+  }
```

Comment:
what is going on here? 
I just need an explanation

Reply:
Without this style, images would tend to grow bigger than any box you put them into.
The logos were growing bigger than their containers, so I added the style. That way, when you specify a width for the logo, the image would actually adjust to fit that style.

Reply:
It is okay. if any issues arise from it. we will solve it. along the way.